quiche/src/lib.rs

```diff
@@ -7401,6 +7401,11 @@ impl Connection {
 
         Ok(pid)
     }
+
+    /// Get the peer's current max number of bidirectional streams.
+    pub fn get_peer_max_streams_bidirectional(&self) -> u64 {
+        self.streams.max_streams_bidi()
+    }
```

Comment:
Sorry I should have explained more clearly. You don't want to do this since the number doesn't do what it sounds like. We should be checking `peer_streams_left_bidi()` which is already available.